src/ast/variables/NamespaceVariable.ts

```diff
@@ -91,25 +91,17 @@ export default class NamespaceVariable extends Variable {
 			return `${t}${safeName}: ${original.getName()}`;
 		});
 
+		if (options.namespaceToStringTag) {
+			members.unshift(`${t}[Symbol.toStringTag]:${_}'Module'`);
+		}
+
 		const name = this.getName();
 
 		const callee = options.freeze ? `/*#__PURE__*/Object.freeze` : '';
 
-		let output = `${options.varOrConst} ${name} = ${
-			options.namespaceToStringTag
-				? `{${n}${members.join(`,${n}`)}${n}};`
-				: `${callee}({${n}${members.join(`,${n}`)}${n}});`
-		}`;
-
-		if (options.namespaceToStringTag) {
-			output += `${n}if${_}(typeof Symbol${_}!==${_}'undefined'${_}&&${_}Symbol.toStringTag)${n}`;
-			output += `${t}Object.defineProperty(${name},${_}Symbol.toStringTag,${_}{${_}value:${_}'Module'${_}});${n}`;
-			output += `else${n || ' '}`;
-			output += `${t}Object.defineProperty(${name},${_}'toString',${_}{${_}value:${_}function${_}()${_}{${_}return${_}'[object Module]'${
-				options.compact ? ';' : ''
-			}${_}}${_}});${n}`;
-			output += `${callee}(${name});`;
-		}
+		let output = `${options.varOrConst} ${name} = ${`${callee}({${n}${members.join(
```

Comment:
For the compact output, there are still a few empty spaces that can be replaced with `${_}`, which will be replaced with an empty string for `compact: true` output (around the `=`). Also, is the template string inside the template string intended?

Reply:
But I see the `${_}` were already missing in the original form.

Reply:
Yeah, nested template string is intended and were there already. You are right,  `${_}` were missing but should I update?

Reply:
Please do!

Reply:
Also if getting rid of the nested template string is possible, that would be nice. Looks like this to me.